src/evaluatorfrontend.cpp

```diff
@@ -67,18 +74,82 @@ void EvaluatorFrontend::FuncRunner() {
 		Plot();
 }
 
-//! @file
-// [runeval]
-void EvaluatorFrontend::RunEvaluator() {
-	while (!evaluator->IsFinished()) {
+int EvaluatorFrontend::EvaluatorFunc() {
```

Comment:
Why does this return an int instead of a bool?

Reply:
Because I'm used to using int instead of bool, but I will change it

Reply:
Once a C programmer, always a C programmer